communitydelphi.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for Delphi
 homepageUrl=https://github.com/integrated-application-development/sonar-delphi
-archivedVersions=1.0.0,1.1.0
-publicVersions=1.2.0
+archivedVersions=1.0.0,1.1.0,1.2.0
+publicVersions=1.3.0
 
 defaults.mavenGroupId=au.com.integradev.delphi
 defaults.mavenArtifactId=sonar-delphi-plugin
 
+1.3.0.description=Improved Delphi 12 support, parsing improvements, bug fixes
+1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.date=2024-03-01
+1.3.0.changelogUrl=https://github.com/integrated-application-development/sonar-delphi/releases/tag/v1.3.0
+1.3.0.downloadUrl=https://github.com/integrated-application-development/sonar-delphi/releases/download/v1.2.0/sonar-delphi-plugin-1.3.0.jar
```

Comment:
This is 404 for me

Reply:
Oops, copy-paste error.
That's fixed now, thanks.